src/Symfony/Component/Validator/Constraints/NotSuspicious.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\Exception\LogicException;
+
+/**
+ * @Annotation
+ *
+ * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
+ *
+ * @author Mathieu Lechat <mathieu.lechat@les-tilleuls.coop>
+ */
+#[\Attribute(\Attribute::TARGET_PROPERTY | \Attribute::TARGET_METHOD | \Attribute::IS_REPEATABLE)]
+class NotSuspicious extends Constraint
+{
+    /**
+     * Check that a string satisfies the requirements for the specified restriction level
+     * (by default {@see \Spoofchecker::HIGHLY_RESTRICTIVE}).
+     */
+    public const CHECK_RESTRICTION_LEVEL = 16;
+
+    /**
+     * Check a string for the presence of invisible characters such as zero-width spaces,
+     * or character sequences that are likely not to display such as multiple occurrences of the same non-spacing mark.
+     */
+    public const CHECK_INVISIBLE = \Spoofchecker::INVISIBLE;
+
+    /** Check a string contains only characters allowed by the configured profile. */
+    public const CHECK_CHAR_LIMIT = \Spoofchecker::CHAR_LIMIT;
+
+    /**
+     * Check that a string does not mix numbers from different numbering systems;
+     * for example “8” (Digit Eight) and “৪” (Bengali Digit Four).
+     */
+    public const CHECK_MIXED_NUMBERS = 128;
+
+    /**
+     * Check that a string does not have a combining character following a character in which it would be hidden;
+     * for example “i” (Latin Small Letter I) followed by a U+0307 (Combining Dot Above).
+     */
+    public const CHECK_HIDDEN_OVERLAY = 256;
+
+    public $message = 'This value is suspicious.';
+    public int $checks = self::CHECK_RESTRICTION_LEVEL | self::CHECK_INVISIBLE | self::CHECK_CHAR_LIMIT | self::CHECK_MIXED_NUMBERS | self::CHECK_HIDDEN_OVERLAY;
+    public int $restrictionLevel = \Spoofchecker::HIGHLY_RESTRICTIVE;
+    public array $profileLocales = [];
+    public bool $addCurrentLocaleToProfile = true;
+
+    public function __construct(
+        array $options = null,
+        string $message = null,
+        int $checks = null,
```

Comment:
Should not be plural since we don't have the ability to combine checks. But maybe could be a good thing to have an array here to be able to combine some checks?

Reply:
`$checks` is a bitmask (as in [`Spoofchecker::setChecks`](https://www.php.net/manual/en/spoofchecker.setchecks.php) so you can indeed combine them, like it is done for the default value.